### PR TITLE
removed card-body and card-width class

### LINKED ISSUE #2720: Remove non-existent classes from citizen-engagement-cards.html
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
As developers, we should make sure our code does not use SCSS classes that do not exist. For this issue, we will remove the non-existent classes `card-body` and `card-width` from a line of code.

### Action Items
- [x] On [line 3 in _includes/citizen-engagement-cards.html](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/_includes/citizen-engagement-cards.html#L3), remove the classes `card-body` and `card-width` from
```
                   <div class="page-card card-primary page-card-lg card-body card-width program-area" data-dropdown>
```
so the result is
```
                   <div class="page-card card-primary page-card-lg program-area" data-dropdown>
```
- [ ] Using Docker locally on your computer, check [Citizen Engagement page](https://www.hackforla.org/citizen-engagement) looks the same as the page on the current site in various desktop, tablet, and mobile views using developer tools

### Resources/Instructions
- [line 3 in _includes/citizen-engagement-cards.html](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/_includes/citizen-engagement-cards.html#L3)
- [Citizen Engagement page](https://www.hackforla.org/citizen-engagement)

### PR DESCRIPTION
**Pull request by scarter31191**
Fixes #2720

### What changes did you make and why did you make them?

  - Removed unused classes card-body and card-width 
  -
  -

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

The unused classes created No visual changes to the website.

### CODE REVIEW

**Comment by riddle015** [comment]
- Availability: 2 hours
- Review ETA: Wed, Mar 16 2022

**Comment by Garcia1416** [comment]
- Availability: 4 hours
- Review ETA: March 14, 2022 or March 15, 2022

File: _includes/citizen-engagement-cards.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% assign visible_projects = site.projects | where: "program-area", "Citizen Engagement" | where: "visible", "true" %}
              {% for item in visible_projects %}
-                   <div class="page-card card-primary page-card-lg card-body card-width program-area" data-dropdown>
+                   <div class="page-card card-primary page-card-lg  program-area" data-dropdown>
```
Comment (JessicaLucindaCheng) [inline _includes/citizen-engagement-cards.html]: I think there are 2 spaces between page-card-lg and program-area. Please remove one of the spaces.

**Review by JessicaLucindaCheng — CHANGES_REQUESTED** [review]
@scarter31191 Thanks for working on this issue!

- [ ] Please make the changes as requested in my in-line comment.

- [ ] You wrote, "Moving files to another directory. No visual changes to the website." No files were moved for this issue, so I'm confused as to to the line "Moving files to another directory" as the reason for no visual changes. I think the reason for no visual changes would be since the scss classes card-body and card-width never existed, removing those classes from the html would not impact the styling of the page. So, please clarify/change "Moving files to another directory. No visual changes to the website."

- [ ] When you are ready for me to re-review your pull request, please click the following in the Reviewers section on the right:
image

**Review by JessicaLucindaCheng — APPROVED** [review]
@scarter31191 Thanks for making the changes! Looks good to me now.

**Pull request merged**